pkg/http/server.go

```diff
@@ -42,6 +42,7 @@ func NewServer(opts Opts) (Server, error) {
 	s := &server{
 		handler: opts.Handler,
 	}
+
```

Comment:
Do we need this change?

Reply:
Probably not, it was probably just my IDE doing some useless formatting.